src/pages/todoApp.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import Todolist from '../components/todoList';
+
+const TodoApp = () => {
+// set initial state
+  const [todo, setTodo] = useState({});
+
+  // get saved todos form the localStorage
+  const savedTodos = () => JSON.parse(localStorage.getItem('todos')) || [];
+
+  // set a state for the todos
+  const [newTodos, setNewTodos] = useState(savedTodos);
+
+  // save todos to the localStorage
+  const updateTodos = (newTodos) => {
+    localStorage.setItem('todos', JSON.stringify(newTodos));
+  };
+
+  // update todos to storage whenever the app state changes
+  updateTodos(newTodos);
+
+  const handleChange = (e) => {
+    setTodo({
+      id: newTodos.length,
```

Comment:
[OPTIONAL] I think it would be better if we use a random Id rather than arrays length, you can use UUID package to generate random ids.
as you can see below two items are getting the same Id.

![code-review-screenshot](https://user-images.githubusercontent.com/97521557/183937736-22195c6a-d2f5-46e5-936b-c58edc23df3f.png)